### DISCUSSION TITLE
Install library in Android Studio

### DISCUSSION THREAD

**Discussion by adrianarrais** [psambit9791/jDSP#56] [Q&A]
Hi!
I added the .jar file from javadoc.io to my android studio project. Now I want to import some classes to my code, what command should I use? 
(p.e. import com.github.psambit9791.jdsp.filter is not working)

**Comment by psambit9791** [comment]
Sorry for the really late response.

Please use Maven/Gradle plugins - that is the easiest way to import the library. Also, javadoc JAR file does not contain the source code to the project - refer to this [link](https://stackoverflow.com/questions/54371447/what-are-the-differences-between-javadoc-jar-sources-jar-and-jar#:~:text=The%20javadoc.,in%20the%20Java%20source%20files.) for details.
